packages-exp/auth-exp/src/api/index.ts

```diff
@@ -62,17 +62,33 @@ export async function performApiRequest<T, V>(
 ): Promise<V> {
   const errorMap = { ...SERVER_ERROR_MAP, ...customErrorMap };
   try {
-    const body = request
-      ? {
+    let body = {};
+    const params: { [key: string]: string } = {
+      key: auth.config.apiKey
+    };
+    if (request) {
+      if (method === HttpMethod.GET) {
+        Object.assign(params, request);
+      } else {
+        body = {
           body: JSON.stringify(request)
-        }
-      : {};
+        };
+      }
+    }
+
+    const queryString = Object.keys(params)
+      .map(
+        key => `${encodeURIComponent(key)}=${encodeURIComponent(params[key])}`
```

Comment:
nit: consider turning this into a standard closure to fix the formatting
```
.map(key => {
  return ...;
}).join('&')
```

Reply:
fixed